modules/sdk/project-templates/project-templates/src/main/java/com/liferay/project/templates/internal/util/FileUtil.java

```diff
@@ -207,7 +207,7 @@ public static Path getFile(Path dirPath, String glob, String regex)
 	public static Path getJarPath() throws URISyntaxException {
 		URI jarUri = _getJarUri();
 
-		return Paths.get(jarUri.getPath());
+		return Paths.get(jarUri);
```

Comment:
@petershin it would be great if we could add a SF rule for this.  Here is the summary.  

Here is the incorrect way to go from URI to Path
```
URI uri = _getURI();

Path path = Paths.get(uri.getPath());
```
Here is correct way

```
URI uri = _getUri();

Path path = Paths.get(uri);
```

How could we contribute a SF rule for this?

Reply:
@gamerson - Hugo is the lead dev for the SF tool.  He'll know the best way to submit SF requests and/or contribute patches.